### DISCUSSION TITLE
[Open source] ESP3D wireless module

### DISCUSSION THREAD

**Discussion by liqijian101** [luc-github/ESP3D#828] [Show and tell]
I made an ESP3D wireless module. It has the function of camera. Its wireless transmission speed is very fast，about 500KB/s.

I have opened it on JLC.
https://oshwhub.com/liqijian101/esp3d_esp32__camer-v2
There is also a test video.
https://www.bilibili.com/video/BV1XD4y1t7kZ/?spm_id_from=333.999.0.0&vd_source=fb6ffe9208ca9ec9b2dfe1301811b6a4

**Comment by luc-github** [comment]
nice,

**Comment by pagabee** [comment]
Thank you for your idea!
You use only one CH440 as switch because you switch only the SPI mode wires.

**Reply by luc-github** [reply]
@liqijian101 is your board on aliexpress?
is there a way to order it from out of china ? what would be the price ? I am interested to test it
please drop me a mail at `luc(at)tech-hunters(dot)com`

**Comment by luc-github** [comment]
@liqijian101 I have linked your module page http://esp3d.io/esp3d/v3.x/documentation/sdtransfer/index.html
If you have english project page let me know - I will linked it also

The setup of http://esp3d.io/esp3d/v3.x is just starting, so there is lot of missing information currently - it should look like http://esp3d.io/esp3d/v2.1.x  which is now done

**Reply by luc-github** [reply]
@liqijian101 any update ?
